src/chrome/manifest_additions.json

```diff
@@ -10,8 +10,5 @@
         "16": "img/icon16.png",
         "48": "img/icon48.png"
     },
```

Comment:
This `icons` property needs to go into the master `manifest.json` - not just chrome additions.

![codesy-addon-icon](https://cloud.githubusercontent.com/assets/71928/16364018/897128c6-3ba2-11e6-96e1-8cc26426bc9e.png)